### ISSUE TITLE
Tweeting from quotes

### ISSUE THREAD

**Issue by ainsliebeattie** [transitionnetwork/TNv3.0#33]
I have spoken about this a couple of times very early on.  Is there any way that a tweet link can be put in a quote box so that people can tweet the quote. I would image this is something that would be added to the flexible content box that we can tick. If we tick it, a tweet / FB share button automatically comes up at the bottom of the quote.

**Comment by yvonnestruthers** [comment]
@thisisyoke @arhinostorm could you give a time estimate for the implementation and testing of this please?

**Comment by ainsliebeattie** [comment]
I don't believe this is an enhancement as it is something that was requested right at the very beginning when we started to discuss the icons and how they might work. I provided examples of this functionality working months and months ago.

**Comment by thisisyoke** [comment]
Hi Ainslie. We did discuss this, I remember having a call with you and Sam and as part of the call we looked through an example and I recall saying it was a great idea but would have to go into a second phase as it was not part of the initial scope / PID. It was never agreed to be implemented in this first phase unfortunately. We would be happy to add it in the future or as part of a change request.
